src/components/Apps/AppInfo/AppInfo.tsx

```diff
@@ -28,6 +33,7 @@ export const AppInfo: FC<AppInfoProps> = (props) => {
       })
       .catch((_) => {
         // use fallback icon if image for id doesn't exist
+        // TODO: error handling
```

Comment:
In the future we could add a link to an existing issue to `TODO`s.  
Helps to tackle all usages later on when that ticket is being worked on.

Reply:
Good idea!